tests/db/conftest.py

```diff
@@ -23,63 +27,31 @@ def restore_config(request, node_config):
 def setup_database(request, node_config):
     print('Initializing test db')
     db_name = node_config['database']['name']
-    get_conn().repl()
+    conn = get_conn()
+
+    if r.db_list().contains(db_name).run(conn):
+       r.db_drop(db_name).run(conn)
```

Comment:
You can run the [pep8](https://pypi.python.org/pypi/pep8) tool,

```
$ pep8 tests/db/conftest.py
```

and see the indentation "error":
 
```bash
tests/db/conftest.py:33:8: E111 indentation is not a multiple of four
```